src/sentry/monitors/endpoints/organization_monitor_details.py

```diff
@@ -186,22 +186,24 @@ def put(self, request: Request, organization, project, monitor) -> Response:
         # Update alert rule after in case slug or name changed
         if "alert_rule" in result:
             # Check to see if rule exists
-            alert_rule = monitor.get_alert_rule()
+            issue_alert_rule = monitor.get_issue_alert_rule()
             # If rule exists, update as necessary
-            if alert_rule:
-                alert_rule_id = update_alert_rule(
-                    request, project, monitor, alert_rule, result["alert_rule"]
+            if issue_alert_rule:
+                issue_alert_rule_id = update_issue_alert_rule(
+                    request, project, monitor, issue_alert_rule, result["alert_rule"]
                 )
             # If rule does not exist, create
             else:
-                alert_rule_id = create_alert_rule(request, project, monitor, result["alert_rule"])
+                issue_alert_rule_id = create_issue_alert_rule(
+                    request, project, monitor, result["alert_rule"]
```

Comment:
anywhere we're referencing `alert_rule` within a response or databody i'm leaving as is